### PR TITLE
Several small improvements to the replica README

### PR DESCRIPTION
**Pull request by rhcarvalho**
- Do not reference inexistent .json file
- Update docs of env vars

@bparees @jhadvig PTAL

### CODE REVIEW

File: 2.4/examples/replica/README.md [hunk 1]
```diff
@@ -2,20 +2,28 @@
...
 
-To create MongoDB replica set in OpenShift v3, you can use the example template
-included in this repository:
+To create a MongoDB replica set in OpenShift v3, you can use the example template
+included in this repository and:
 
-* `oc create -f ./2.4/examples/replica/mongodb-clustered.json`
+* add a new template to the current project:
+```
+$ oc create -f 2.4/examples/replica/mongodb-clustered.json
```
Comment (bparees) [inline 2.4/examples/replica/README.md]: if they follow these instructions they'll still need to take another action to instantiate the template.  that's not very clear from this.

maybe just delete it and only include the new-app instruction.

Comment (rhcarvalho) [inline 2.4/examples/replica/README.md]: Ok, that's why I added the other one ;)

How about we change the order, `oc new-app` first, `oc create` last? I see the value of `oc create` in that it allows you to work from the Web Console later.

Comment (rhcarvalho) [inline 2.4/examples/replica/README.md]: @bparees eventually I went your way, kept only `oc new-app`.
Would get more verbose to highlight that you can use the Web Console, but not only, as after the template is added you can also use `oc new-app --template`.

**Comment by bparees** [comment]
yeah we have docs to cover "all the ways you can do stuff and why", for our examples providing one good way is sufficient.

[merge]

**Comment by mnagy** [comment]
[merge]

**Pull request merged**